website/conferences/model.py

```diff
@@ -25,6 +25,19 @@ class Conference(StoredObject):
     public_projects = fields.BooleanField(required=False, default=True)
     poster = fields.BooleanField(default=True)
     talk = fields.BooleanField(default=True)
+    field_names = fields.DictionaryField(
```

Comment:
Right- but NEW conferences that we want to customize WILL need to be able to fill in this field, if it is present.